src/dymaptic.GeoBlazor.Core/Components/Renderers/RasterStretchRenderer.cs

```diff
@@ -0,0 +1,144 @@
+﻿using dymaptic.GeoBlazor.Core.Components.Layers;
+using dymaptic.GeoBlazor.Core.Components.Renderers.ColorRamps;
+using dymaptic.GeoBlazor.Core.Extensions;
+using dymaptic.GeoBlazor.Core.Serialization;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace dymaptic.GeoBlazor.Core.Components.Renderers;
+
+/// <summary>
+///     RasterStretchRenderer defines the symbology with a gradual ramp of colors for each pixel in a ImageryLayer, ImageryTileLayer,
+///     and WCSLayer based on the pixel value. The RasterStretchRenderer works well when you have a large range of values to display,
+///     such as in imagery, aerial photographs, or elevation models. Important to note that RasterStretchRenderer does not inherit from
+///     Renderer class.
+///     <a target="_blank" href="https://developers.arcgis.com/javascript/latest/api-reference/esri-renderers-RasterStretchRenderer.html">
+///         ArcGIS
+///         JS API
+///     </a>
+/// </summary>
+/// 
+public class RasterStretchRenderer : LayerObject
+{
+    public RasterStretchRenderer() { }
+
+    /// <inheritdoc />
+    [JsonPropertyName("type")]
+    public string Type => "raster-stretch";
+
+    public MultipartColorRamp? ColorRamp { get; set; }
```

Comment:
Create a second constructor that will set all properties, and use it in the sample when creating this class in C#. Set properties to `private set;`